vignettes/pr_review_guidance.Rmd

```diff
@@ -181,65 +177,57 @@ for (pkg in base_recommended_pkgs) {
 assign(".lib.loc", ".library", envir = environment(.libPaths))
 
 r_version <- getRversion()
-if (grepl("^3.6", r_version)) {
-  options(repos = "https://cran.microsoft.com/snapshot/2020-02-29")
-} else if (grepl("^4.0", r_version)) {
-  options(repos = "https://cran.microsoft.com/snapshot/2021-03-31")
+if ([grepl](https://rdrr.io/r/base/grep.html)("^3.6", r_version)) {
+  [options](https://rdrr.io/r/base/options.html)(repos = "https://cran.microsoft.com/snapshot/2020-02-29")
+} else if ([grepl](https://rdrr.io/r/base/grep.html)("^4.0", r_version)) {
+  [options](https://rdrr.io/r/base/options.html)(repos = "https://cran.microsoft.com/snapshot/2021-03-31")
 } else {
-  options(repos = "https://cran.rstudio.com")
+  [options](https://rdrr.io/r/base/options.html)(repos = "https://cran.rstudio.com")
```

Comment:
This should be in line with https://github.com/pharmaverse/admiralci/blob/main/.github/workflows/r-cmd-check.yml. I.e., at the moment it should handle 4.0, 4.1, and 4.2.
See also https://github.com/pharmaverse/admiralci/issues/26

Reply:
@bundfussr 

- How can one access the highlighted repos noted in the R-CMD-check from the link below

https://github.com/pharmaverse/admiralci/blob/main/.github/workflows/r-cmd-check.yml. 
![reversions](https://user-images.githubusercontent.com/112789549/197002966-cfa6ae6b-00ea-4dbe-9975-49c526c42e8c.png)

- up to `https://packagemanager.rstudio.com/` shows the page below, is that the intended page?

![reversions2](https://user-images.githubusercontent.com/112789549/197003175-a2d66503-bb74-4cdd-b4de-558c97c063c2.png)

- I made the updates to include 4.0, 4.1, and 4.2 in `admiraldev`. but version 4.2 is not included in `r-cmd-check.yml`, would you prefer I remove it from `admiraldev` or create an issue to include it in `admiralci` 

![repos](https://user-images.githubusercontent.com/112789549/197008105-2a46fd58-0422-4c69-ae81-b2adef0aaa4e.png)

https://github.com/pharmaverse/admiralci/blob/main/.github/workflows/r-cmd-check.yml.